system/lib/libc/musl/arch/emscripten/syscall_arch.h

```diff
@@ -30,120 +30,121 @@
-long __syscall102(int which, ...);
-long __syscall104(int which, ...);
-long __syscall114(int which, ...);
-long __syscall118(int which, ...);
-long __syscall121(int which, ...);
-long __syscall122(int which, ...);
-long __syscall125(int which, ...);
-long __syscall132(int which, ...);
-long __syscall133(int which, ...);
-long __syscall140(int which, ...);
-long __syscall142(int which, ...);
-long __syscall144(int which, ...);
-long __syscall145(int which, ...);
-long __syscall146(int which, ...);
-long __syscall147(int which, ...);
-long __syscall148(int which, ...);
-long __syscall150(int which, ...);
-long __syscall151(int which, ...);
-long __syscall152(int which, ...);
-long __syscall153(int which, ...);
-long __syscall163(int which, ...);
-long __syscall168(int which, ...);
-long __syscall178(int which, ...);
-long __syscall180(int which, ...);
-long __syscall181(int which, ...);
-long __syscall183(int which, ...);
-long __syscall191(int which, ...);
-long __syscall192(int which, ...);
-long __syscall193(int which, ...);
-long __syscall194(int which, ...);
-long __syscall195(int which, ...);
-long __syscall196(int which, ...);
-long __syscall197(int which, ...);
-long __syscall198(int which, ...);
-long __syscall199(int which, ...);
-long __syscall200(int which, ...);
-long __syscall201(int which, ...);
-long __syscall202(int which, ...);
-long __syscall203(int which, ...);
-long __syscall204(int which, ...);
-long __syscall205(int which, ...);
-long __syscall207(int which, ...);
-long __syscall208(int which, ...);
-long __syscall209(int which, ...);
-long __syscall211(int which, ...);
-long __syscall212(int which, ...);
-long __syscall218(int which, ...);
-long __syscall219(int which, ...);
-long __syscall220(int which, ...);
-long __syscall221(int which, ...);
-long __syscall252(int which, ...);
-long __syscall265(int which, ...);
-long __syscall268(int which, ...);
-long __syscall269(int which, ...);
-long __syscall272(int which, ...);
-long __syscall295(int which, ...);
-long __syscall296(int which, ...);
-long __syscall297(int which, ...);
-long __syscall298(int which, ...);
-long __syscall299(int which, ...);
-long __syscall300(int which, ...);
-long __syscall301(int which, ...);
-long __syscall302(int which, ...);
-long __syscall303(int which, ...);
-long __syscall304(int which, ...);
-long __syscall305(int which, ...);
-long __syscall306(int which, ...);
-long __syscall307(int which, ...);
-long __syscall308(int which, ...);
-long __syscall320(int which, ...);
-long __syscall324(int which, ...);
-long __syscall330(int which, ...);
-long __syscall331(int which, ...);
-long __syscall333(int which, ...);
-long __syscall334(int which, ...);
-long __syscall337(int which, ...);
-long __syscall340(int which, ...);
-long __syscall345(int which, ...);
+#define EM_SYSCALL_ARGS int arg1, int arg2, int arg3, int arg4, int arg5, int arg6, int arg7
+long __syscall1(EM_SYSCALL_ARGS);
+long __syscall3(EM_SYSCALL_ARGS);
```

Comment:
so this will have 7 args for each of these calls? then the JS versions will not match the # of arguments, slowing things down at the wasm/js boundary in some engines. we should use the proper number in each one here.